src/client/ua_client_highlevel_subscriptions.c

```diff
@@ -5,7 +5,7 @@
 #include "ua_client_highlevel.h"
 #include "ua_client_internal.h"
 #include "ua_util.h"
-
+#include "stdio.h"
```

Comment:
This should not be necessary. Use `UA_LOG_` macros instead of `printf`.
`stdio.h` is a platform specific include and should therefore only be used within the `plugins` folder